openshift_performance/ose3_perf/scripts/build_test_1.py

```diff
@@ -0,0 +1,414 @@
+#!/usr/bin/env python
```

Comment:
Rename to build_test.py as this will be a replacement unless you feel we should keep both around for some period of time.

Reply:
Done